.gitmodules

```diff
@@ -1,9 +1,15 @@
-[submodule "buildres/abbrv.jabref.org"]
+[submodule "abbrv.jabref.org"]
   path = buildres/abbrv.jabref.org
   url = https://github.com/JabRef/abbrv.jabref.org.git
-[submodule "buildres/csl-styles"]
```

Comment:
why the name change?

Reply:
The path itself did not change. The name of the submodule was NOT reflecting the actual path. Since the name is an identifier for humans, I thought, briefty is better. I could also fix the path name, but was afraid that af an update of the path in the future the name would not be updated and thus inconsistent again.